src/cephadm/cephadm

```diff
@@ -3626,7 +3628,10 @@ def command_unit():
     call_throws([
         'systemctl',
         args.command,
-        unit_name])
+        unit_name],
+        verbose=True,
+        desc=''
```

Comment:
```suggestion
        verbose=True
```


nit, but why not allow for `command[0]` to appear in the output?

```
$ cephadm unit --name node-exporter.host3 enable
Inferring fsid 0d848f8c-23ce-11eb-8fe2-3ab7774c4e84
systemctl: stderr Created symlink /etc/systemd/system/ceph-0d848f8c-23ce-11eb-8fe2-3ab7774c4e84.target.wants/ceph-0d848f8c-23ce-11eb-8fe2-3ab7774c4e84@node-exporter.host3.service → /etc/systemd/system/ceph-0d848f8c-23ce-11eb-8fe2-3ab7774c4e84@.service
```

Reply:
hm. 

All those `systemctl:` prefixes are just noise. Especially in this command. I'd love to also remove the `stdout:` prefix, but that was a bit excessive. 

ok?